graphenex/core/web/__init__.py

```diff
@@ -1,11 +1,13 @@
-from graphenex.core.cli.shell import Shell
-from graphenex.core.utils.logcl import GraphenexLogger
+import secrets
+import webbrowser
 
 from flask import Flask
 from flask_socketio import SocketIO
-import webbrowser
-import secrets
 
+from graphenex.core.cli.shell import Shell
+from graphenex.core.utils.logcl import GraphenexLogger
+from graphenex.core.web.views import *
+from graphenex.core.web.providers import *
```

Comment:
You should import these after initializing the logger etc. otherwise we get this error:

```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/home/orhun/gh/grapheneX/graphenex/__main__.py", line 6, in <module>
    from graphenex.core.web import run_server
  File "/home/orhun/gh/grapheneX/graphenex/core/web/__init__.py", line 6, in <module>
    from graphenex.core.web.views import *
  File "/home/orhun/gh/grapheneX/graphenex/core/web/views.py", line 10, in <module>
    from graphenex.core.web import app, logger, socketio
ImportError: cannot import name 'app' from partially initialized module 'graphenex.core.web' (most likely due to a circular import) (/home/orhun/gh/grapheneX/graphenex/core/web/__init__.py)
```

Patch: 

```diff
diff --git a/graphenex/core/web/__init__.py b/graphenex/core/web/__init__.py
index 0b90ddb..5dd1eb0 100644
--- a/graphenex/core/web/__init__.py
+++ b/graphenex/core/web/__init__.py
@@ -6,9 +6,6 @@ from flask_socketio import SocketIO
 
 from graphenex.core.cli.shell import Shell
 from graphenex.core.utils.logcl import GraphenexLogger
-from graphenex.core.web.views import *
-from graphenex.core.web.providers import *
-
 
 logger = GraphenexLogger(__name__)
 app = Flask(__name__)
@@ -17,6 +14,8 @@ app.config['ACCESS_TOKEN'] = secrets.token_urlsafe(6)
 socketio = SocketIO(app)
 default_addr = ('localhost', '8080')
 
+from graphenex.core.web.views import *
+from graphenex.core.web.providers import *
 
 def disable_flask_logs():
     """Disable the Flask logs"""

```

Reply:
My bad, fixed this